### DISCUSSION TITLE
How to disable rewriting `element`'s content with selected date

### DISCUSSION THREAD

**Discussion by cjayyy** [easepick/easepick#20] [Q&A] [answered]
Is there any hidden option to disable rewriting of trigger element's content with selected date value? 
If I want to use the element just as trigger (and position reference) and handle the value rendering somewhere else (via `getDate()`) I don't want the trigger to be rewritten.

**Comment by wakirin** [comment, marked as answer]
If I understand the case correctly, you can do something like this: https://jsfiddle.net/waki/yc1ds9Lm/

**Reply by cjayyy** [reply]
That is correct. I was just hoping it would be possible without need to provide custom trigger logic. My only need is really just to not render selected value in `element`.

**Reply by cjayyy** [reply]
I meant something like this https://github.com/easepick/easepick/pull/21/.

**Comment by oficialsosystems** [comment]
hello, how do I translate the calendar to Brazilian Portuguese that I didn't find in the documentation?

**Reply by wakirin** [reply]
Hello,
https://easepick.com/examples/using-another-language

For your case it will be `pt-BR`. https://jsfiddle.net/waki/srwdej6L/
If you use `RangePlugin` don't forget to provide locale for tooltip: https://easepick.com/packages/range-plugin#option-locale

Note: next time please create a new discussion because this discussion is not related with your question.

**Comment by gaultierq** [comment]
when calendar is inlined, the element remains filled with the selected date. This is a unexpected behavior in my opinion. 
In my case, I could use:
```
                picker.on('select', (event) => {
                    this.elementTarget.innerHTML = ""
                    // custom behavior
                });
```
